cmake/modules/SetUpFreeBSD.cmake

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.
```

Comment:
```suggestion
# Copyright (C) 1995-2023, Rene Brun and Fons Rademakers.
```

Reply:
well, this file is not quite copyright Rene and Fons if we are being strict:) But let's go with it.

Reply:
Appreciated; this is something we're working on addressing.